crates/rpc/eth/filter.rs

```diff
@@ -0,0 +1,165 @@
+use std::time::{SystemTime, UNIX_EPOCH};
+
+use crate::utils::RpcErr;
+use crate::RpcHandler;
+use rand::prelude::*;
+use serde_json::json;
+
+use super::logs::LogsRequest;
+
+#[derive(Debug, Clone)]
+pub struct FilterRequest {
+    pub request_data: LogsRequest,
+}
+
+impl RpcHandler for FilterRequest {
+    fn parse(params: &Option<Vec<serde_json::Value>>) -> Result<Self, crate::utils::RpcErr> {
+        let filter = LogsRequest::parse(params)?;
+        Ok(FilterRequest {
+            request_data: filter,
+        })
+    }
+    fn handle(
+        &self,
+        storage: ethereum_rust_storage::Store,
+    ) -> Result<serde_json::Value, crate::utils::RpcErr> {
+        let filter = self.request_data.request_to_filter(&storage)?;
+        let id: u64 = random();
```

Comment:
Could you investigate geth and see if they use a random key or a deterministic one? 

As an example, see how they generate the payload id: https://github.com/lambdaclass/ethereum_rust/pull/540/files#diff-4446e6639c57085d9923271c8e951db627558bd79daefc708351b4ccd2281e53R29

Reply:
Sure, I'll doble check this.

Reply:
Just for the record, the ids are random:
- [filter subscription definition](https://github.com/ethereum/go-ethereum/blob/82b0dec7135b281c1b03064d50959dc992c2f94f/eth/filters/filter_system.go#L240-L245).
- [RPC id type definition](https://github.com/ethereum/go-ethereum/blob/15fb0dcc6732f3b04f8d3b13e309afd723364ba5/rpc/subscription.go#L50-L58)